### ISSUE TITLE
Update to GitHub GraphQL API v4 to optimize network bandwidth

### ISSUE THREAD

**Issue by velias** [Daskiworks/ghwatch#91]
Update to GitHub GraphQL API v4 if it provides support for notifications - it should allow better control what informations are returned in responses, so lower a network bandwidth substantially (as currently used v3 REST API returns lots of nonimportant info in responses of /notification API)

**Comment by laughedelic** [comment]
I think this is still missing: https://platform.github.community/t/notifications-and-events-apis-seem-to-be-missing/3114

**Comment by velias** [comment]
Yep, I did API investigation and realized it also which is a pity :-( Thanks for linking helpful article. I'll try to contact github support to push this topic a bit, but I was generally unlucky when I tried it previously.
